app/cdap/components/Administration/TetheringTabContent/OdfTetheringConnections/Connections/index.tsx

```diff
@@ -14,18 +14,19 @@
  * the License.
  */
 
-import React, { useState } from 'react';
-import styled from 'styled-components';
+import React from 'react';
 import T from 'i18n-react';
 import ConnectionsTable from './ConnectionsTable';
 import { HeaderContainer, HeaderTitle, BodyContainer, NoDataText } from '../../shared.styles';
-import { IConnection } from '../types';
+import { IConnection } from '../../types';
 
 const PREFIX = 'features.Administration.Tethering';
 const I18NPREFIX = `${PREFIX}.Connections`;
 
 interface IConnectionsProps {
   connections: IConnection[];
+  handleEdit: (connType: string, peer: string) => void;
+  handleDelete: (connType: string, peer: string) => void;
```

Comment:
These props will be passed down to the appropriate child component (i.e., ConnectionsTable) in a subsequent PR where established connections are handled (for both cdf and odf).